components/chart/src/selection-area/selection-area.ts

```diff
@@ -724,14 +724,20 @@ export class DtChartSelectionArea implements AfterContentInit, OnDestroy {
     // –––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––
     // Decide weather to show the range or a timestamp according to a click or a drag.
     // On a mousedown the range and the timestamp have to be hidden.
+    //
+    // The pipe delay is used in case that both the start and hide listening for the _click$
+    // so the start has to be delayed a little bit that it will be destroyed a few milliseconds
+    // before it gets created again.
```

Comment:
Where is the delay pipe? Can't find one in the changeset :D

Reply:
ah old comment solved it without :D